Makefile

```diff
@@ -15,6 +15,12 @@
 #-----------------------------------------------------------------------------
 # Global Variables
 #-----------------------------------------------------------------------------
+
+# Invoke make with DEBUG=<anything> will have the binaries built with debugger information
+ifdef DEBUG
```

Comment:
I'd prefer if DEBUG=0 can be used to explicitly opt-out of debugging.  I know that Costin mentioned a preference to debug by default (and this preference is certainly consistent with other products I've worked on), but regardless of the default I think it'd be ideal for the release build to have an explicit opt-out.  I suppose this would be either:
ifneq ($(DEBUG),0)
or
ifeq ($(DEBUG),1)
depending on what the default ends up being.

Reply:
I remember he mentioned that too. I can make 'debug' as default.

Let me check on explicit values 0 & 1 for DEBUG

Reply:
You'll also want to be concerned about the release builds, which is handled by release/cloud_builder.sh.  I'm thinking the most straightforward thing is to add an "export DEBUG=0", perhaps near where the other instances of "export" exist in that file.  This will help cover steps that invoke scripts instead of make (I've got pending PRs to migrate most or all of those to make), while cutting down on the need to add "DEBUG=0" to all the invocations of make.

Reply:
@matdelco thanks for the reminder. will do.